Tasks/AzureCLIV1/azureclitask.ts

```diff
@@ -125,23 +126,23 @@ export class azureclitask {
             let authType: string = tl.getEndpointAuthorizationParameter(connectedService, 'authenticationType', true);
             let cliPassword: string = null;
             var servicePrincipalId: string = tl.getEndpointAuthorizationParameter(connectedService, "serviceprincipalid", false);
+            var tenantId: string = tl.getEndpointAuthorizationParameter(connectedService, "tenantid", false);
+
             if (authType == "spnCertificate") {
```

Comment:
this.tenantId can be set even for spnCertificate based auth-type

Reply:
Yes, I opted to keep the same semantics since servicePrincipalId was only being set on SPN (and the docs state that is the semantic). 

If you think it makes sense I can make a change to set both servicePrincipalId and TenantId on SPN cerficiations

Reply:
I think it would make sense to add tenantId and serviceprincipalId both  - given the help message says that these are available when auth scheme is service principal

Reply:
Done, it now exports servicePrincipalId and TenandId if it's a certificate and servicePrincipalId ,servicePrincipalKey and tenantId if it's a service principal